ogr/ogrsf_frmts/miramon/ogrmiramonlayer.cpp

```diff
@@ -788,6 +788,8 @@ OGRFeature *OGRMiraMonLayer::GetFeature(GIntBig nFeatureId)
                 // Get X,Y (z). MiraMon has no multipoints
                 if (MMGetGeoFeatureFromVector(phMiraMonLayer, nIElem))
                 {
+                    CPLError(CE_Failure, CPLE_AppDefined,
```

Comment:
CPLError() messages shouldn't start or end with a newline character. Applies to below instances as well

Reply:
Sure. Corrected. Sorry I didn't pay attention to that.
Some lost files added. It's checking now